src/preferences.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self'; script-src 'unsafe-inline'">
+  <link rel="stylesheet" href="./assets/css/style.min.css">
+  <title>Preferences</title>
```

Comment:
This title is overriding the `title` set in `app.js`, we should be able to safely remove this.